packages/utils/telemetry-utils/src/test/errorLogging.spec.ts

```diff
@@ -337,53 +338,74 @@ describe("Error Logging", () => {
 			assert.strictEqual(props.foo, undefined, "foo should have been omitted");
 			assert.strictEqual(props.bar, "normal", "bar should not be omitted");
 		});
-		it("addTelemetryProperties - adds to object, returned from getTelemetryProperties, overwrites", () => {
+		it("addTelemetryProperties - adds to object, returned from getTelemetryProperties, doesn't overwrite", () => {
 			const loggingError = new LoggingError("myMessage", { p1: 1, p2: "two", p3: true });
 			(loggingError as any).p1 = "should not be overwritten";
 			loggingError.addTelemetryProperties({
 				p1: "ignored",
 				p4: 4,
-				p5: { value: 5, tag: "CodeArtifact" },
+				p5: true,
+				p6: { value: 5, tag: "CodeArtifact" },
```

Comment:
Add a tagged array or record for coverage?

Reply:
The typing doesn't support an array of records, it wants `(string | number | boolean)[]`, but I added the test case for a tagged array with all of those types.